src/Utilities/Request.php

```diff
@@ -194,10 +194,10 @@ public function columnKeyword(int $index): string
     protected function prepareKeyword(float|array|int|string $keyword): string
     {
         if (is_array($keyword)) {
-            return implode(' ', $keyword);
+            $keyword = implode(' ', $keyword);
         }
 
-        return (string) $keyword;
+        return e((string) $keyword);
```

Comment:
casting into a string is no longer needed as `e()`function return a string
```suggestion
        return e($keyword);
```

Reply:
I just recalled that the cast was to pass phpstan since the keyword param also accepts int & float values. https://github.com/yajra/laravel-datatables/actions/runs/3519589634/jobs/5899694603